src/store/reducers/devicesReducer.js

```diff
@@ -157,6 +170,10 @@ export const getDevicesError = state =>
   getError(getDevicesReducer(state), epics.actionTypes.fetchDevices);
 export const getDevicesPendingStatus = state =>
   getPending(getDevicesReducer(state), epics.actionTypes.fetchDevices);
+export const getConditionedDevicesError = state =>
```

Comment:
Are devices conditioned? Like, are they out of shape or what?  Why not just name this the same way as its corresponding things "...DevicesByCondition..."?